config/locales/en.yml

```diff
@@ -382,11 +382,11 @@ en:
         success: Successfully updated group %{namespace_name} with group %{group_name} link %{param_name}
     workflow_execution_actions:
       cancel:
-        error: Could not cancel workflow %{workflow_name}
-        success: Workflow %{workflow_name} was successfully canceled
+        error: Could not cancel pipeline %{workflow_name}
```

Comment:
**nitpick (blocking):** The following 4 changes should be reverted.

Reply:
Updated in 51f582d